### ISSUE TITLE
Feat: Add a "Feedback Form/Page with button to Website

### ISSUE THREAD

**Issue by MastanSayyad** [Sanchitbajaj02/palettegram#371]
### What feature?

Hey @Sanchitbajaj02 

I would like to add a **feedback form/page**

To improve user experience from visitors, i suggest to add a feedback form/page to the website. This form will allow users to provide their opinions, report issues, and suggest improvements directly.

### Add screenshots

- **For example like this:**

![image](https://github.com/Sanchitbajaj02/palettegram/assets/101971980/a54ebc07-ed10-4ddd-b09c-06b3f95eab30)

**Please assign this issue to me.**

### Record

- [X] I agree to follow this project's Code of Conduct
- [X] I want to work on this issue

**Comment by Sanchitbajaj02** [comment]
Hi @MastanSayyad, the feedback form is missing, and it has to be there. 
Make sure to code the backend logic as well for this feature. Do contact me if you need help in Appwrite

**Comment by MastanSayyad** [comment]
Hey @Sanchitbajaj02 I have created the feedback form and added into the footer but Im unable to setup routes for it, can you pls tell me how and where to do that? as I have more experience in reactjs than nextjs

![image](https://github.com/Sanchitbajaj02/palettegram/assets/101971980/7f9c802c-7e0e-4eb6-b9b5-0ab854833bc1)


![image](https://github.com/Sanchitbajaj02/palettegram/assets/101971980/11d5f1a0-4dfa-4da8-982f-7d4733e29278)

**Comment by Sanchitbajaj02** [comment]
> Hey @Sanchitbajaj02 I have created the feedback form and added into the footer but Im unable to setup routes for it, can you pls tell me how and where to do that? as I have more experience in reactjs than nextjs
> 
> ![image](https://private-user-images.githubusercontent.com/101971980/339968755-7f9c802c-7e0e-4eb6-b9b5-0ab854833bc1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg0NTE3NDUsIm5iZiI6MTcxODQ1MTQ0NSwicGF0aCI6Ii8xMDE5NzE5ODAvMzM5OTY4NzU1LTdmOWM4MDJjLTdlMGUtNGViNi1iOWI1LTBhYjg1NDgzM2JjMS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNjE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDYxNVQxMTM3MjVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wOWQ1ZDJhM2ZkYjgxNjJhOWVjNjIyNzg4NDNkNTMzODNkNmYxOWNmMGM5MTFjYTZlYzRiNTU3NjI3ZDFmMzRmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.9WHTje4GBCgSU0jwxCj7LfeN_6DS1GbmTbOnukUTxmk)
> 
> ![image](https://private-user-images.githubusercontent.com/101971980/339968469-11d5f1a0-4dfa-4da8-982f-7d4733e29278.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg0NTE3NDUsIm5iZiI6MTcxODQ1MTQ0NSwicGF0aCI6Ii8xMDE5NzE5ODAvMzM5OTY4NDY5LTExZDVmMWEwLTRkZmEtNGRhOC05ODJmLTdkNDczM2UyOTI3OC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNjE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDYxNVQxMTM3MjVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02MTU4YjEyYjgzY2UxN2RjNmQ3ZWVjMzhmYzYwNWNmNDg1ZGRkOTFjNjNhYzYyMTI3YjI5YzIyNzViNzdiOGI3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.M3cfuF5IJg-IbTOszgEeVQ1xZCaVmw6OobQH7_fAzYU)

Hey @MastanSayyad 
what you have to do is create a page.tsx file in location src/app/feedback/page.tsx. Now create a server component and call your component in it.

**Comment by MastanSayyad** [comment]
@Sanchitbajaj02 Yes I did that too but still after clicking feedback in the footer it doesn't navigate
- page.tsx

![image](https://github.com/Sanchitbajaj02/palettegram/assets/101971980/3af8aa4c-120a-4bec-9cb8-178d55df0148)

- footer

![image](https://github.com/Sanchitbajaj02/palettegram/assets/101971980/230dd170-17fa-4f91-8ee6-b4c322061915)

**Comment by Sanchitbajaj02** [comment]
Are you able to go to that page manually by filling in url?

**Comment by MastanSayyad** [comment]
Yeah, sure, It would be a **Big help!**
I'll DM you on Discord. However, I will need to do so later as it is currently raining here, and there is a power outage. Additionally, my system is not charged.

**Comment by Sanchitbajaj02** [comment]
> Yeah, sure, It would be a **Big help!** I'll DM you on Discord. However, I will need to do so later as it is currently raining here, and there is a power outage. Additionally, my system is not charged.

Don't worry, you can DM whenever you are comfortable

**Comment by Sanchitbajaj02** [comment]
@MastanSayyad what is the status? If i didn't get the response in 24hrs, I am unassigning this from you

**Comment by MastanSayyad** [comment]
@Sanchitbajaj02 I tried solving this but I am still stuck with the routing issue, you can unassign me, extremely sorry for the inconvenience from my side!

**Comment by SuperSahitya** [comment]
I can work on this issue. Just please explain whether I just use Email.js or do you want this form to only be accessible by authorized user ?

**Comment by Sanchitbajaj02** [comment]
> I can work on this issue. Just please explain whether I just use Email.js or do you want this form to only be accessible by authorized user ?

Hi @SuperSahitya, it will be saved into the database and only authorized users can access it
